garage/sampler/base.py

```diff
@@ -1,40 +1,8 @@
-import numpy as np
+from abc import ABC
 
-from garage.logger import logger, tabular
-from garage.misc import special, tensor_utils
-from garage.sampler import utils
 
-
-class Sampler:
-    def start_worker(self):
-        """
-        Initialize the sampler, e.g. launching parallel workers if necessary.
-        """
-        raise NotImplementedError
-
-    def obtain_samples(self, itr):
```

Comment:
why remove all these abstract methods from Sampler? this seems to be the sampler interface.